app/models/concerns/file_set/geo_file_format_behavior.rb

```diff
@@ -38,20 +30,23 @@ def vector_work?
   end
 
   module ClassMethods
-    def image_file_formats
-      ['TIFF', 'IMAGE_FILE']
-    end
-
-    def raster_file_formats
-      ['TIFF_GeoTIFF', 'RASTER_FILE']
+    def gdal_formats
+      [
+        'image/tiff; gdal-format=GTiff',
+        'text/plain; gdal-format=AIGrid'
+      ].freeze
     end
 
-    def vector_file_formats
-      ['SHAPEFILE', 'SHAPEFILE_ZIP', 'VECTOR_FILE']
+    def ogr_formats
+      [
+        'application/zip; ogr-format=ESRI_Shapefile',
```

Comment:
I think we should keep the format values as-is in the OGR/GDAL command line API. So this would be `ogr-format="ESRI Shapefile"`, right?